src/lib/bootloader/bootloader_base.rb

```diff
@@ -36,7 +36,7 @@ def prepare
     end
 
     # writes configuration to target disk
-    def write; end
+    def write(etc_only: false); end
```

Comment:
```suggestion
    # @param etc_only [Boolean] true on transactional systems
    #   because /boot is read-only there
    def write(etc_only: false); end
```

Reply:
thanks